cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowStatsSlaHttpHandlerTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.app.services.http.handlers;
+
+import org.junit.Test;
+
+/**
+ * Tests for {@link co.cask.cdap.gateway.handlers.WorkflowStatsSlaHttpHandler}
+ */
+public class WorkflowStatsSlaHttpHandlerTest {
+
+  @Test
+  public void testStatistics() {
+
```

Comment:
This might be a controversial topic, but I'm going to have to agree with @gsps1. That is a good idea.